src/ai/backend/manager/api/gql/artifact.py

```diff
@@ -741,6 +743,9 @@ class ArtifactRevisionImportTask:
 class ImportArtifactsPayload:
     artifact_revisions: ArtifactRevisionConnection
     tasks: list[ArtifactRevisionImportTask]
+    errors: list[BackendAIError] = strawberry.field(
+        default_factory=list, description="Errors that occurred during import artifacts"
+    )
```

Comment:
It seems wrong to include the name "BackendAIError." There is too much room for misunderstanding.